gas/config/tc-riscv.c

```diff
@@ -2410,6 +2492,91 @@ riscv_ip (char *str, struct riscv_cl_insn *ip, expressionS *imm_expr,
 		}
 	      break;
 
+            case 'n':
+	      {
+		char field_name[MAX_KEYWORD_LEN];
+		args++;
+		if (parse_nds_v5_field (&args, field_name))
+		  {
+		    if (strcmp (field_name, "nds_rc") == 0
+			&& reg_lookup (&s, RCLASS_GPR, &regno))
+		      {
+			INSERT_OPERAND (RC, *ip, regno);
+			args--;
+			continue;
+		      }
+		    else if (strcmp (field_name, "nds_rdp") == 0
+			     && reg_lookup (&s, RCLASS_GPR, &regno))
+		      {
+			if (xlen == 32 && (regno % 2) != 0)
+			  {
+			    as_bad (_("The number of Rd must be even "
+				      "(limitation of register pair)."));
+			    break;
+			  }
+			INSERT_OPERAND (RD, *ip, regno);
+			args--;
+			continue;
+		      }
+		    else if (strcmp (field_name, "nds_rsp") == 0
+			     && reg_lookup (&s, RCLASS_GPR, &regno))
+		      {
+			if (xlen == 32 && (regno % 2) != 0)
+			  {
+			    as_bad (_("The number of Rs1 must be even "
```

Comment:
It would be great if the error/warning messages are starting with lower case letter, and without the period at the end.

Reply:
Thanks for pointing it out. Fixed it the latest commit.